core/src/main/java/com/netease/arctic/table/LocationKind.java

```diff
@@ -0,0 +1,4 @@
+package com.netease.arctic.table;
+
+public interface LocationKind {
```

Comment:
As I see, a static final string is enough, why using interface?

Reply:
Because need hive module to extend, "hive" can not in core